lib/inc/drogon/plugins/Hodor.h

```diff
@@ -130,16 +130,16 @@ class DROGON_EXPORT Hodor : public drogon::Plugin<Hodor>
     size_t limiterExpireTime_{600};
     std::function<optional<std::string>(const drogon::HttpRequestPtr &)>
         userIdGetter_;
-    std::function<HttpResponsePtr(const HttpRequestPtr &)>
+    std::function<HttpResponsePtr(const drogon::HttpRequestPtr &)>
         rejectResponseFactory_;
 
-    void onHttpRequest(const HttpRequestPtr &,
+    void onHttpRequest(const drogon::HttpRequestPtr &,
                        AdviceCallback &&,
                        AdviceChainCallback &&);
-    bool checkLimit(const HttpRequestPtr &req,
+    bool checkLimit(const drogon::HttpRequestPtr &req,
                     const LimitStrategy &strategy,
                     const std::string &ip,
-                    const optional<std::string> &userId);
+                    const std::optional<std::string> &userId);
```

Comment:
please use drogon::optional here, drogon use boost::optional when compiler only supports c++14.
see https://github.com/drogonframework/drogon/blob/master/lib/inc/drogon/utils/optional.h for more details.

Reply:
Without these changes, the framework was not compiled for Win. And this is first project where I'm not using boost)

Reply:
![image](https://user-images.githubusercontent.com/10835602/224378762-5a8dd59c-c54b-43ec-b5ae-444580965365.png)

Reply:
Maybe need some trick. I have not ability check with boost and without C14
```c
namespace drogon
{
#if __cplusplus >= 201703L || (defined _MSC_VER && _MSC_VER > 1900)
	#define DROGON_OPTINAL	std::optional
#else
	#define DROGON_OPTINAL	boost::optional
#endif
}  // nested namespaces  in drogon
```

Reply:
try to use drogon::optional and drogon::string_view

Reply:
done. Compiled on MSVC2015 C++14